umbra-js/src/types.ts

```diff
@@ -115,3 +115,9 @@ export interface UserAnnouncement {
   token: string; // token address
   txHash: string;
 }
+
+export interface SendBatch {
+  address: string;
+  token: string;
+  amount: string;
```

Comment:
When you have a type that expects user-provided inputs (e.g. token amounts), you typically want to take a `BigNumberish` so the API is more flexible and easier to use, then the function that consumes this `amount` should `BigNumber.from(amount)` to parse it
```suggestion
  amount: BigNumberish;
```

OTOH, when returning a value from a function, `BigNumber` is what you want (or `bigint` in a more modern codebase)